.github/scripts/monday/removeLabel.js

```diff
@@ -16,14 +16,18 @@ module.exports = async ({ context }) => {
   const { labels: issueLabels, assignee } = issue;
   const [labelName] = assertRequired([label?.name]);
 
-  if (labelName === spike && issueLabels) {
-    const isSpikeComplete = issueLabels.some((label) => label.name === spikeComplete);
-    assertRequired([isSpikeComplete], "Issue is marked as a spike complete. Skipping label removal.");
+  if (labelName === spike && issueLabels && includesLabel(issueLabels, spikeComplete)) {
+    console.log("Issue is marked as a spike complete. Skipping label removal.");
+    process.exit(0);
   }
 
-  if ([designTokens, tokensPackage].includes(labelName) && issueLabels) {
-    const areTokensStillPresent = issueLabels.some((label) => label.name === designTokens);
-    assertRequired([areTokensStillPresent], "Issue is still marked as a design token issue. Skipping label removal.");
+  const tokensLabels = [designTokens, tokensPackage];
+  if (tokensLabels.includes(labelName) && issueLabels) {
+    const remainingLabel = tokensLabels.find((label) => label !== labelName);
+    if (issueLabels.some((label) => label.name === remainingLabel)) {
```

Comment:
Could this use `includesLabel`? If so, it would remove the check for `issueLabels` above.